### PR TITLE
feat(frontend): added new formatReceiveOutMinimun util

### PR DESCRIPTION
**Pull request by roman-nazaruk**
# Motivation

The logic for calculating and formatting the minimum expected receive amount after applying slippage was previously duplicated and embedded inside components. This PR introduces a dedicated utility function to centralize that logic, improve code reuse, and increase overall clarity and maintainability.

# Changes

- Added `formatReceiveOutMinimum` utility:
  - Accepts `slippageValue`, `receiveAmount`, and `decimals`
  - Calculates slippage-adjusted amount
  - Formats the result based on token precision
- Simplified related component logic by delegating formatting to the new utility

# Tests

- Added unit tests for `formatReceiveOutMinimum`
  - Covers valid numeric and string input
  - Handles edge cases like empty, null, and undefined slippage
  - Validates formatting and calculation behavior

### CODE REVIEW

File: src/frontend/src/lib/utils/swap.utils.ts [hunk 1]
```diff
@@ -120,3 +124,33 @@ export const calculateSlippage = ({
...
+ * Formats the minimum expected receive amount after applying slippage.
+ *
+ * @param {OptionAmount} params.slippageValue - The slippage percentage (as string or number or undefined).
+ * @param {number} params.receiveAmount - The original quoted amount to receive.
+ * @param {number} params.decimals - The number of decimal places for the token.
+ * @returns {AmountString | null} A formatted string representing the minimum amount the user can expect to receive,
+ * formatted with the token's decimals, or `null` if any required value is missing.
+ *
+ */
+export const formatReceiveOutMinimum = ({
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/utils/swap.utils.ts:138]: shall this maybe go in the `format.utils` module? or do you prefer it here? no strong opinion, just asking

File: src/frontend/src/lib/utils/swap.utils.ts [hunk 2]
```diff
@@ -120,3 +124,33 @@ export const calculateSlippage = ({
...
+	slippageValue,
+	receiveAmount,
+	decimals
+}: FormatSlippageParams): AmountString | undefined => {
+	if (isNullishOrEmpty(slippageValue?.toString())) {
+		return undefined;
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/utils/swap.utils.ts:144]:
```suggestion
		return;
```

**Review by AntonioVentilii — APPROVED** [review]
LGTM tks!

File: src/frontend/src/lib/utils/swap.utils.ts (hunk 1 above)
Comment (roman-nazaruk) [reply src/frontend/src/lib/utils/swap.utils.ts:138]: TBH, I would leave it here as this util method can be reused only for swaps
